model/src/main/java/org/cloudfoundry/identity/uaa/zone/UserConfig.java

```diff
@@ -50,6 +52,7 @@ public boolean allGroupsAllowed() {
     }
 
     // return defaultGroups plus allowedGroups
+    @Nullable
```

Comment:
When using this annotation, the IDE is able to show warnings when using the return value without a preceding null check. I tend to use this annotation in cases like this where the return type is a collection and the method returns null instead of an empty collection.